archive/frames/tests/test_views.py

```diff
@@ -299,21 +299,41 @@ def test_filters_public(self):
             content_type='application/json'
         )
         self.client.force_login(user)
-        proposal_frame = FrameFactory(public_date=datetime.datetime(2999, 1, 1, tzinfo=UTC), proposal_id='prop1')
+        proposal_proprietary_frame = FrameFactory(public_date=datetime.datetime(2999, 1, 1, tzinfo=UTC), proposal_id='prop1')
+        proposal_public_frame = FrameFactory(public_date=datetime.datetime(1992, 11, 14, tzinfo=UTC), proposal_id='prop1')
         public_frame = PublicFrameFactory()
 
+        # If public=false, then a logged in user should see all of their own data, proprietary or not
```

Comment:
Could we add a test where the query `public` is not provided? I think you mentioned that the fronted always passes this in explicitly so it probably doesn't matter, but I always get confused what the default behavior is when you don't pass it.

Reply:
Done - it basically defaults to showing public data. I think this is sensible.